src/bun.js/webcore/streams.zig

```diff
@@ -54,53 +54,52 @@ pub const ReadableStream = struct {
 
     pub const Strong = struct {
         held: JSC.Strong = .{},
-        self: ?ReadableStream = null,
+        globalThis: ?*JSGlobalObject = null,
 
-        pub fn init(this: ReadableStream, globalThis: *JSGlobalObject) Strong {
+        pub fn init(this: ReadableStream, globalThis: *JSGlobalObject) !Strong {
             switch (this.ptr) {
                 .Blob => |stream| {
-                    stream.parent().ref_count += 1;
+                    try stream.parent().incrementCount();
                 },
                 .File => |stream| {
-                    stream.parent().ref_count += 1;
+                    try stream.parent().incrementCount();
                 },
                 .Bytes => |stream| {
-                    stream.parent().ref_count += 1;
+                    try stream.parent().incrementCount();
                 },
                 else => {},
             }
             return .{
-                .self = this,
+                .globalThis = globalThis,
                 .held = JSC.Strong.create(this.value, globalThis),
             };
         }
 
         pub fn get(this: *Strong) ?ReadableStream {
-            if (this.held.has()) {
-                return this.self;
+            if (this.globalThis) |globalThis| {
+                if (this.held.get()) |value| {
+                    return ReadableStream.fromJS(value, globalThis);
```

Comment:
this did not work as expected for some reason

Reply:
Looks like server does `stream.detach` so thats why was not working, we cannot detach anymore